auraed/src/cri/error.rs

```diff
@@ -0,0 +1,70 @@
+/* -------------------------------------------------------------------------- *\
+ *        Apache 2.0 License Copyright © 2022-2023 The Aurae Authors          *
+ *                                                                            *
+ *                +--------------------------------------------+              *
+ *                |   █████╗ ██╗   ██╗██████╗  █████╗ ███████╗ |              *
+ *                |  ██╔══██╗██║   ██║██╔══██╗██╔══██╗██╔════╝ |              *
+ *                |  ███████║██║   ██║██████╔╝███████║█████╗   |              *
+ *                |  ██╔══██║██║   ██║██╔══██╗██╔══██║██╔══╝   |              *
+ *                |  ██║  ██║╚██████╔╝██║  ██║██║  ██║███████╗ |              *
+ *                |  ╚═╝  ╚═╝ ╚═════╝ ╚═╝  ╚═╝╚═╝  ╚═╝╚══════╝ |              *
+ *                +--------------------------------------------+              *
+ *                                                                            *
+ *                         Distributed Systems Runtime                        *
+ *                                                                            *
+ * -------------------------------------------------------------------------- *
+ *                                                                            *
+ *   Licensed under the Apache License, Version 2.0 (the "License");          *
+ *   you may not use this file except in compliance with the License.         *
+ *   You may obtain a copy of the License at                                  *
+ *                                                                            *
+ *       http://www.apache.org/licenses/LICENSE-2.0                           *
+ *                                                                            *
+ *   Unless required by applicable law or agreed to in writing, software      *
+ *   distributed under the License is distributed on an "AS IS" BASIS,        *
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+ *   See the License for the specific language governing permissions and      *
+ *   limitations under the License.                                           *
+ *                                                                            *
+\* -------------------------------------------------------------------------- */
+
+use aurae_client::AuraeClientError;
+use thiserror::Error;
+use tonic::Status;
+use tracing::error;
+
+pub(crate) type Result<T> = std::result::Result<T, RuntimeServiceError>;
+
+#[derive(Debug, Error)]
+pub enum RuntimeServiceError {
+    #[error("sandbox '{sandbox_id}' already exists")]
+    SandboxExists { sandbox_id: String },
+    #[error("sandbox '{sandbox_id}' not found")]
+    SandboxNotFound { sandbox_id: String },
+    #[error("Failed to kill sandbox '{sandbox_id}': {error}")]
```

Comment:
I know it's weird wording sometimes but I tend to like the error messages to start the same. "sandbox {sandbox_id}...". Only a preference